src/components/AudioTrack/AudioTrack.vue

```diff
@@ -13,28 +17,35 @@
       {{ audio.category }}
     </div>
 
-    <div class="waveform-section flex flex-row gap-2">
+    <div class="flex flex-row gap-2">
       <PlayPause
         v-if="isCompact"
         class="flex-shrink-0"
         :is-playing="isPlaying"
         :disabled="!isReady"
         @toggle="setPlayerState"
       />
-      <Waveform
-        :class="isCompact ? 'h-20' : 'h-30'"
-        :is-ready="isReady"
-        :current-time="currentTime"
-        :duration="duration"
-        :peaks="audio.peaks"
-        :show-duration="isCompact"
-        @seeked="setPosition"
-      />
+      <div
+        class="waveform-section"
```

Comment:
This class should not be a 'section', as the section is the level of div above it

- `info-section`
- `interactive-section`
  - &lt;this div&gt;
    - Waveform
- `info-section`